docs/source/ux_limitations.rst

```diff
@@ -179,3 +179,45 @@ JAX supports transforming over
 using special control flow operators (e.g. ``jax.lax.cond``, ``jax.lax.while_loop``).
 We're investigating adding equivalents of those to functorch
 (open an issue on `GitHub <https://github.com/pytorch/functorch>`_ to voice your support!).
+
+Randomness
+----------
+Note that like with JAX's PRNG system, our system only determine the randomness behavior of PyTorch
+operators
+
+Our randomness differs from JAX because we aren't using a stateless PRNG, in part because PyTorch doesn't
+have full support for a stateless PRNG. Instead, we've introduced a flag system to allow for the most
+common forms of randmoness that we see. If your use case does not fit these forms of randomness, please
+file an issue.
```

Comment:
I feel like people reading this might not know what JAX's PRNG system is, so leading with it without discussing what it is might be confusing. There's a couple of main point I think we want to get across:
1. randomness inside vmap is tricky
2. we allow the user to control random PyTorch operations only (e.g. we cannot control python randomness, numpy randomness)
3. How to use the randomness API
4. Discussion of how this is different or similar to JAX's system
Would it make sense to move the discussion on JAX to the end?

Reply:
Makes sense! I added a little intro and moved this to the end.